pkg/v1/tkg/kind/client.go

```diff
@@ -366,6 +373,12 @@ func (k *KindClusterProxy) getKindNetworkingConfig() kindv1.Networking {
 		default:
 			serviceSubnet = constants.DefaultIPv4ServiceCIDR
 		}
+	} else if ipFamily == constants.DualStackPrimaryIPv6Family {
+		// We expect the IPv4,IPv6 order for the CIDR in both cases for KinD until
+		// we bump to a version of the kind library that includes a fix for this issue:
+		// https://github.com/kubernetes-sigs/kind/issues/2484
+		serviceSubnets := strings.Split(serviceSubnet, ",")
+		serviceSubnet = fmt.Sprintf("%s,%s", serviceSubnets[1], serviceSubnets[0])
```

Comment:
Please factor out `serviceSubnet` calculation into its own method:
```go
serviceSubnet := k.serviceSubnet(ipFamily)
```

In that method, please avoid using `else if` (e.g. by returning from the `if` body).

When you're done, the body of `getKindNetworkingConfig()` should become linear!

Reply:
Done.